src/main/java/org/opensearch/ad/cluster/ADDataMigrator.java

```diff
@@ -1,301 +1,301 @@
-// @anomaly-detection.create-detector Commented this code until we have support of Job Scheduler for extensibility
-// *
-// * SPDX-License-Identifier: Apache-2.0
-// *
-// * The OpenSearch Contributors require contributions made to
-// * this file be licensed under the Apache-2.0 license or a
-// * compatible open source license.
-// *
-// * Modifications Copyright OpenSearch Contributors. See
-// * GitHub history for details.
-// */
-//
-// package org.opensearch.ad.cluster;
-//
```

Comment:
Do we really need to comment this code? 
If we have to track the changes `main` or git history on feature/extensions should help. 
I would remove it.

Reply:
Didn't get you. I have actually uncommented the entire ADDataMigrator. Am I missing anything?

Reply:
Nvm my bad. I interpreted it the other way.